TypeCobol.Analysis/Graph/IMultiBranchContext.cs

```diff
@@ -0,0 +1,51 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace TypeCobol.Analysis.Graph
+{
+    /// <summary>
+    /// Multi Branch Context interface for Block that participate to a branching in the Control Flow Graph.
+    /// </summary>
+    public interface IMultiBranchContext<N, D>
```

Comment:
Why the interface ? I mean it describes the public API of the MultiBranchContext but we could simply make the class itself public keeping all members we don't want to expose with internal visibility.
The interface itself is never used directly.

Reply:
using this interface allows to have readonly properties outside, and thus it reduces the amount of code to be changed.

Reply:
See my remark in `MultiBranchContext.cs`, actually not creating the interface would result in less code changes.

Reply:
We can keep the interface to avoid moving/exposing the `BasicBlockForNode` class.